exporter/exporterhelper/concurrency_sender.go

```diff
@@ -0,0 +1,89 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package exporterhelper // import "go.opentelemetry.io/collector/exporter/exporterhelper"
+
+import (
+	"context"
+	"errors"
+	"sync"
+
+	"go.uber.org/multierr"
+	"go.uber.org/zap"
+	"golang.org/x/sync/semaphore"
+
+	"go.opentelemetry.io/collector/component"
+	"go.opentelemetry.io/collector/exporter"
+)
+
+const defaultNumSenders = 10
+
+type ConcurrencySettings struct {
+	Enabled    bool `mapstructure:"enabled"`
```

Comment:
This option seems excessive. It's the same as if concurrency is set to any number other than `0`.

We probably don't even need a configuration grouping here. Just one field in the exporter config similar to timeout, e.g.:

```
otlp:
  timeout: 10s
  concurrency: 100
  sending_queue:
   enabled: true
  retry_on_failure:
    enabled: true
```

Reply:
Thanks, simplified my config as you suggested